src/pages/EventDetailsPage/components/DeleteEditButtons/index.js

```diff
@@ -30,26 +32,28 @@ const DeleteEditButtons = props => {
 
   const confirmButtonProps = {
     name: 'Yes',
-    onClick: handleConfirmDelete,
-    to: ROUTES.CALENDAR,
-    component: Link,
+    actionCallback: () => {
```

Comment:
see this is the point i'm trying to make, I'm putting a Delete button, so why do i have to put my delete function inside a confirmButtonProps thingy when im used to pasing it into onclick